packages/babel-types/scripts/generators/docs.js

```diff
@@ -173,107 +262,47 @@ function generateMapAliasToNodeTypes() {
-  Miscellaneous:
-    "A cover of non-standard AST types that are sometimes useful for development.",
-  ModuleDeclaration:
-    "A cover of ImportDeclaration and [ExportDeclaration](#exportdeclaration)",
-  ModuleSpecifier:
-    "A cover of import and export specifiers. Note: It is _not_ the [ModuleSpecifier](https://tc39.es/ecma262/#prod-ModuleSpecifier) defined in the spec.",
-  ObjectMember:
-    "A cover of [members](https://tc39.es/ecma262/#prod-PropertyDefinitionList) in an object literal.",
-  Pattern:
-    "A cover of [BindingPattern](https://tc39.es/ecma262/#prod-BindingPattern) except Identifiers.",
-  PatternLike:
-    "A cover of [BindingPattern](https://tc39.es/ecma262/#prod-BindingPattern)s. ",
-  Private: "A cover of private class elements and private identifiers.",
-  Property: "A cover of object properties and class properties.",
-  Pureish:
-    "A cover of AST nodes which do not have side-effects. In other words, there is no observable behaviour changes if they are evaluated more than once.",
-  Scopable:
-    "A cover of [FunctionParent](#functionparent) and [BlockParent](#blockparent).",
-  Standardized:
-    "A cover of AST nodes which are part of an official ECMAScript specification.",
-  Statement:
-    "A cover of any [Statement](https://tc39.es/ecma262/#prod-Statement)s.",
-  TSBaseType: "A cover of primary TypeScript type annotations.",
-  TSEntityName: "A cover of ts entities.",
-  TSType: "A cover of TypeScript type annotations.",
-  TSTypeElement: "A cover of TypeScript type declarations.",
-  TypeScript: "A cover of AST nodes defined for TypeScript.",
-  Terminatorless:
-    "A cover of AST nodes whose semantic will change when a line terminator is inserted between the operator and the operand.",
-  UnaryLike: "A cover of UnaryExpression and SpreadElement.",
-  UserWhitespacable: "Deprecated. Will be removed in Babel 8.",
-  While:
-    "A cover of DoWhileStatement and WhileStatement, which share the same AST shape.",
-};
+
 const mapAliasToNodeTypes = generateMapAliasToNodeTypes();
 readme.push("### Aliases");
 readme.push("");
 for (const alias of [...mapAliasToNodeTypes.keys()].sort()) {
   const nodeTypes = mapAliasToNodeTypes.get(alias);
   nodeTypes.sort();
   if (!(alias in aliasDescriptions)) {
-    throw new Error(
-      'Missing alias descriptions of "' +
-        alias +
-        ", which covers " +
-        nodeTypes.join(",")
-    );
+    if (alias in t.DEPRECATED_ALIASES) {
+      const newAlias = t.DEPRECATED_ALIASES[alias];
+      let aliasDescription = `Deprecated, use [\`${newAlias}\`](#${newAlias.toLowerCase()}) instead.`;
+      if (alias in aliasDeprecationNotes) {
+        aliasDescription += " " + aliasDeprecationNotes[alias];
+      }
+      // wrap alias description in caution admonitions
+      // https://docusaurus.io/docs/markdown-features/admonitions
+      aliasDescription = `:::caution\n\n${aliasDescription}\n\n:::`;
+      aliasDescriptions[alias] = aliasDescription;
```

Comment:
Here is what it look like:

https://deploy-preview-2729--babel.netlify.app/docs/babel-types#moduledeclaration